packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -335,14 +335,6 @@ class HotRunner extends ResidentRunner {
     }
 
     final Stopwatch restartTimer = new Stopwatch()..start();
-    if (previewDart2) {
```

Comment:
Can you add a TODO here that you would have to add the reset back once the restart logic is switched back to using the incremental kernel generator.

Reply:
Done